### ISSUE TITLE
SAC Discrete needs it's own `calculate_entropy_tuning_losses` function?

### ISSUE THREAD

**Issue by Harimus** [p-christ/Deep-Reinforcement-Learning-Algorithms-with-PyTorch#75]
So while checking the [SAC_Discrete](https://github.com/p-christ/Deep-Reinforcement-Learning-Algorithms-with-PyTorch/blob/master/agents/actor_critic_agents/SAC_Discrete.py#L64) code I noticed the lack of `calculate_entropy_tuning_losses` function, which it inherit from [SAC](https://github.com/p-christ/Deep-Reinforcement-Learning-Algorithms-with-PyTorch/blob/master/agents/actor_critic_agents/SAC_Discrete.py#L64). 

But according the [SAC_Discrete paper](https://arxiv.org/pdf/1910.07207.pdf) equation 11 vs 9 (latter is for continuous SAC), for the discrete case, the Estimate _E_ is rather taken by weighting the `-alpha*(log_pi + target_entropy)` with the probability of each action by the agent. ( `pi`), and not by sampling one `log_pi`.

Shouldn't `SAC_Discrete` have it's own entropy loss function then?
